### ISSUE TITLE
[Accessibility]A11y_ Java Tooling for IntelliJ/Eclipse_ New Dynamic Web App_Screen Reader: NVDA is reading the list items inside the dropdown twice

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/azure-tools-for-java#3329]
[Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #C+E-H; #Java tooling for IntelliJ/Eclipse; #NVDA

**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0

**OS Details:** 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: NVDA
MAS Violated: MAS1.3.1

**Repro Steps:** 
1) Start NVDA and Open Java EE - Eclipse application 
2) Install 'Azure Toolkit for Eclipse' from Marketplace settings/preferences popup
3) Tab till tools-->Azure-->Azure Sign in and sign into Azure account and select any subscription
4) Tab till File-->New-->Project--->Dynamic Web Project
5) Tab till any dropdown present in the popup
6) Verify if NVDA is reading the list items inside the dropdown twice

**Actual Result:** 
NVDA is reading the list items inside the dropdown twice

**Expected Result:** 
NVDA should read the list items inside the dropdown only once

**User Impact:** 
Screen Reader users will find it confusing and difficult to understand if NVDA is reading the list items inside the dropdown twice

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_10_NVDA is reading the list items inside the dropdown twice](https://user-images.githubusercontent.com/41109135/62204685-58aca600-b3ab-11e9-8e5c-cf6c805184f8.png)

**Comment by Flanker32** [comment]
This UI belong to Eclipse IDE, which is out of the scope of our toolkit, we have submitted an issue in Eclipse tracking [here](https://bugs.eclipse.org/bugs/show_bug.cgi?id=550725)

**Comment by Flanker32** [comment]
- Importance of experience: Normal, users need to do selection in many situation, where they may meet this issue.
- Quality of experience: Low, users could know the value of selected item, but it could be confused when screen reader read it twice.
- Alternate solutions: No alternate technologies or options can be leveraged.
- Evidence: All evidence is provided in this issue and external issue.

**Comment by Flanker32** [comment]
@Tanushagottipati Please help us review this issue.

**Comment by Flanker32** [comment]
Tracking in AAT [here](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/56452990-0d38-45bc-b475-a62832ce0076), @Tanushagottipati please have a review.

**Comment by Tanushagottipati** [comment]
External Bug: https://bugs.eclipse.org/bugs/show_bug.cgi?id=550725

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario.

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749746
